python/ray/tune/BUILD

```diff
@@ -59,7 +59,7 @@ py_test(
     size = "large",
     srcs = ["tests/test_api.py"],
     deps = [":tune_lib"],
-    tags = ["team:ml", "exclusive", "rllib"],
+    tags = ["team:ml", "exclusive", "rllib", "new_storage"],
```

Comment:
nit/for future reference - we don't need this if we're enabling the FF (or is this test run in multiple suites?)

Reply:
These tests are disabled in the old path but enabled in the new path. When the FF is enabled for everything (and not overwritten) we don't need it anymore.

Let's clean those up in one go.

Reply:
Ack, just to confirm my understanding - these tests are run in other suites in addition to `":octopus: :brain: Tune tests and examples {using RLlib}"`?